Misc/NEWS.d/next/Core and Builtins/2021-07-19-19-53-46.bpo-44676.WgIMvh.rst

```diff
@@ -1,2 +1,2 @@
-Add ability to serialise ``types.Union`` objects. Patch provided by Yurii
+Add ability to serialise ``types.UnionType`` objects. Patch provided by Yurii
```

Comment:
We should not change this news entry :stuck_out_tongue:

Reply:
Didn't know about it 😕

Reply:
Oh wait, maybe we do actually want this changed as it hasn't been released yet :facepalm:

Reply:
no problem.
Can you please tell me X time in deprecation warning?

Reply:
I don't know, that's up to the core devs, ask in https://bugs.python.org/issue44732 if we should have a deprecation period and if so, how long.

Reply:
Okey thanks 😄